src/script/Context.mjs

```diff
@@ -0,0 +1,123 @@
+import {
+  LUA_REGISTRYINDEX,
+  lua_call,
+  lua_createtable,
+  lua_gc,
+  lua_insert,
+  lua_isstring,
+  lua_pcall,
+  lua_pushcclosure,
+  lua_pushstring,
+  lua_rawgeti,
+  lua_setglobal,
+  lua_settop,
+  lua_tolstring,
+  luaL_loadbuffer,
+  luaL_newstate,
+  luaL_openlibs,
+  luaL_ref,
+  to_jsstring,
+  to_luastring,
+} from './lua';
```

Comment:
An alternative to these massive import statements would be to export an object instead, and have usage become: `lua.lua_pcall`. Thoughts @fallenoak?

Reply:
How about something like `import * as lua from './lua';`?

Reply:
On second thought, perhaps that would be a performance hit, having to lookup these methods on `lua` for each call?

Reply:
Maybe the first time, but once the optimizer kicks in, I doubt it?

Reply:
Hm, good point. This might warrant some testing. OK to merge as-is and iterate?

Reply:
Fine by me.